pages/api/level/[id].ts

```diff
@@ -62,6 +62,21 @@ export default withAuth(async (req: NextApiRequestWithAuth, res: NextApiResponse
       },
     });
   
+    const level = await LevelModel.findOne({
```

Comment:
I think this query isn't needed because you already have the level id

Reply:
The function returns a level though (line 65), would it break anything to not have it?